RELEASE_NOTES

```diff
@@ -1,5 +1,9 @@
 Release Notes for rogueRegister EPICS module
 
+pre-release: <DATE> <AUTHOR>
```

Comment:
I didn't mean literally `<date>` and `<author> `... Can you please replace with:
```
pre-release:   2025-07-16  Jeremy Lorelli
```